backend/scripts/init_db_from_scraping.py

```diff
@@ -0,0 +1,163 @@
+    if class_dict["teacher_name"]=="TBD":
+        class_dict["teacher_name"]=['FÉVRIER', 'Philippe']
+    if class_dict["teacher_name"]==['ZERBIB', 'Olivier', 'David']:
+        class_dict["teacher_name"]=['ZERBIB', 'Olivier']
+
+
+for class_dict in classes:
+    if class_dict["teacher_name"] not in teacher_names:
+        teacher_names.append(class_dict["teacher_name"])
+# print(teacher_names)
+
+
+
+# teacher_names = [
+#     ("Antoine", "Chancel"), 
+#     ("Laurent", "Linnemer"), 
+#     ("Olivier", "Loisel"), 
+#     ("Philippe", "Choné"),
+#     ("Philippe", "Février"), 
+# ]
+
+
+def initialize_database():
+    with app.app_context():
+        db.create_all()
+        
+        student_ids = []
+        teacher_ids = []
+        class_ids = []
+        lesson_ids = []
+        
+        # 1 - Add students
+        print("Adding students...")
+        for (name, surname) in student_names:
+            student_ids.append(
+                add_user(
+                    name=name,
+                    surname=surname,
+                    profile_pic=f"photos/{name.lower()}_{surname.lower()}.jpg",
+                    user_type="student",
+                    level=str(random.randint(1, 3))+"A",
+                )
+            )
+        
+        # 2 - Add teachers
+        print("Adding teachers...")
+        # print(teacher_names)
+        for complete_name in teacher_names:
+            name = complete_name[0]
+            surname = complete_name[1]
+            teacher_ids.append(
+                add_user(
+                    name=name,
+                    surname=surname,
+                    profile_pic=f"photos/{name.lower()}_{surname.lower()}.jpg",
+                    user_type="teacher",
+                )
+            )
+        
+        # 3 - Add classes
+        print("Adding classes...")
+        for class_dict in classes:
+            class_ids.append(
+                add_class(
+                    class_id=class_dict["class_id"],
+                    name=class_dict["name"],
+                    ects_credits=class_dict["ects"],
+                    default_teacher_id=get_user_by_name(*class_dict["teacher_name"]).user_id,
+                    backgroundColor=class_dict["backgroundColor"],
+                )
+            )
+        
+        # 4 - Authorize teachers for classes
+        print("Authorizing teachers for classes...")
+        for class_dict in classes:
+            # for teacher in class_dict["teacher_name"]:
+            #     print(teacher)
+            # authorize_teacher_for_class(get_user_by_name(*teacher).user_id, class_dict["class_id"])
+            authorize_teacher_for_class(get_user_by_name(*class_dict["teacher_name"]).user_id, class_dict["class_id"])
+                
+        # 5 - Enroll students in classes
+        print("Enrolling students in classes...")
+        for student_id in student_ids:
+            for class_id in class_ids:
+                if random.random() < 0.7:
```

Comment:
Maybe reduce this one to 0.1 - 0.2, too much classes per students now
<img width="1728" alt="image" src="https://github.com/thomktz/betterave/assets/60552083/1a1579d8-8c2f-4320-a562-926861cc3be6">